src/__tests__/assets.test.ts

```diff
@@ -97,8 +97,10 @@ describe('assets/', () => {
       }
 
       describe('networkAddresses', () => {
-        it('should contain the same networks in all files', async () => {
-          const files = versionFiles(version);
+        it('should contain the same networks in all files (exception for migration contracts)', async () => {
```

Comment:
nit: We should not ignore these files, but verify that they are a strict subset of the other files.

In particular, you should not be able to have chain FOO in `safe_migration.json` if it is not in `safe.json`.

Reply:
Wrote a new test for this case 👍🏾